### PR TITLE
Community page

### LINKED ISSUE #787: Add community page for docs

### PR DESCRIPTION
**Pull request by cicdw**
Closes #787

### CODE REVIEW

File: docs/guide/welcome/community.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 0
+---
+
+# Community
+
+Prefect welcomes issues, contributions and discussion from all users regardless of background or experience level.  [cicdw](https://github.com/cicdw) is the main point of contact for the Prefect repository.
```
Comment (jlowin) [inline docs/guide/welcome/community.md]:
```suggestion
We welcome issues, contributions and discussion from all users, regardless of background or experience level. Please consider [Chris White](https://github.com/cicdw) is the main point of contact for the Prefect repo.
```

File: docs/guide/welcome/community.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- for realtime discussion or help, please visit our [Gitter chat](https://gitter.im/prefectio/prefect)
+- important architechtural and design decisions are memorialized in [Prefect Improvement Notices (PINs)](../PINs/PIN-1-Introduce-PINs.html)
+
+
+## Contributing
+
+Prefect encourages users to contribute in any way they can!  
+
+There are a number of ways you can get started contributing:
+- **issues**: the [issue board](https://github.com/PrefectHQ/prefect) contains issues with the tag ["good first issue"](https://github.com/PrefectHQ/prefect/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) which should be approachable for first time contributors
```
Comment (jlowin) [inline docs/guide/welcome/community.md]: I expected this section titled `**issues**` to be about "opening issues to highlight problems". I'd call this one either `**closing issues**` or `**contributing code**` and have a separate header called "issues" for literally issues.

Comment (cicdw) [reply docs/guide/welcome/community.md]: Updated; I agree we need a "How to open issues" section, but wasn't sure whether it was worthwhile to tackle right now

File: docs/guide/welcome/community.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+sidebarDepth: 0
+---
+
+# Community
+
+We welcome issues, contributions and discussion from all users, regardless of background or experience level. Please consider [Chris White](https://github.com/cicdw) is the main point of contact for the Prefect repo.
```
Comment (jlowin) [inline docs/guide/welcome/community.md]:
```suggestion
We welcome issues, contributions and discussion from all users, regardless of background or experience level. Please consider [Chris White](https://github.com/cicdw) as the main point of contact for the Prefect repo.
```
Sorry!

Comment (cicdw) [reply docs/guide/welcome/community.md]: No worries!

**Pull request merged**